packages/webapp/src/components/Table/TableV2.jsx

```diff
@@ -24,7 +24,7 @@ import TableContainer from '@mui/material/TableContainer';
 import TablePagination from '@mui/material/TablePagination';
 import TableRow from '@mui/material/TableRow';
 import { BsThreeDots } from 'react-icons/bs';
-import clsx, { ClassValue } from 'clsx';
```

Comment:
I was testing adding this to prop types and for some reason it breaks when unused? Weird stuff.

Reply:
It's probably because in a jsx file it expects ClassValue to be a JS value but it's actually a TS type and so it's not able to resolve the import.